### DISCUSSION TITLE
How do I recover my educational github Account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#28599] [GitHub Education]
Hi, I have a email with my university (Santiago de Chile), in some ocassion i cant login with this account and github have banned my educational account. what can I do, and skiss me for my english.
![image](https://user-images.githubusercontent.com/72051806/183262051-6cb914d7-0c61-4413-b37a-652891a9c5d4.png)

**Comment by jfullstackdev** [comment]
hopefully this will be seen by the Staff here,

other than that, you may want to try,

https://support.github.com/contact/account

I noticed also, it's in Accessibility category, it should be GitHub Education
